### DISCUSSION TITLE
Upgrade guide to keep up to date with templates

### DISCUSSION THREAD

**Discussion by HurrellT** [callstack/react-native-builder-bob#515] [Ideas]
I would love to have an upgrade guide to keep old libraries up to date not only with dependencies but also the template.
I have a js library built two years ago and I created a new project to test changes but it's completely different, starting with the use of expo in the latest version of crnl.
I'll migrate the lib to the new version but I would love to keep these up to date.
Thanks!
Please let me know if I can help with anything to make this possible.

**Comment by satya164** [comment]
There is plan to have a upgrade helper like for regular React Native apps https://github.com/react-native-community/discussions-and-proposals/pull/721

**Reply by HurrellT** [reply]
Sounds awesome! I'll collab in there. Thanks!
